packages/plugin-table/src/TableCellDomObjectRenderer.ts

```diff
@@ -72,8 +71,8 @@ export class TableCellDomObjectRenderer extends NodeRenderer<DomObject> {
      * @param selector
      * @param commandId
      */
-    _mapButton(referenceCell: TableCellNode, deleteTable: HTMLElement, commandId: string): void {
-        deleteTable.addEventListener(
+    _mapButton(referenceCell: TableCellNode, target: HTMLElement, commandId: string): void {
+        target.addEventListener(
```

Comment:
I don't understand what `deleteTable` meant, but `target` is very generic. If it's always supposed to be a table, why not call it `table` ? If it's always a cell why not `cell` ? Or `domTable` / `domCell` if you prefer.

Reply:
`button` I guess. This is to put an event listener on a button.
I wouldn't spend any time on this right now though because @Goaman was commissioned to make a new UI for this, made it, and then FP asked for a 4th version of it instead ^^

Reply:
:skull: I have removed the whole function in my branch :skull:

Reply:
Lol we know, that's why that line was removed.